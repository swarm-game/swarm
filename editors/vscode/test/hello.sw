// SYNTAX TEST "swarm" "sample testcase"

def hello: Text = "Hello World" end;

pure hello;
