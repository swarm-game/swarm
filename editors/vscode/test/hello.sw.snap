>// SYNTAX TEST "swarm" "sample testcase"
#^^ source.swarm comment.line.double-slash
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.swarm comment.line.double-slash
>
>def hello: Text = "Hello World" end;
#^^^^ source.swarm keyword.control.dictionary.def
#    ^^^^^ source.swarm keyword.control.dictionary.def entity.name.function
#         ^ source.swarm keyword.control.dictionary.def keyword.control.dictionary.def.annotation
#          ^ source.swarm keyword.control.dictionary.def
#           ^^^^ source.swarm keyword.control.dictionary.def entity.name.type
#               ^ source.swarm keyword.control.dictionary.def
#                ^ source.swarm keyword.control.dictionary.def keyword.control.dictionary.def.assignment
#                 ^ source.swarm
#                  ^ source.swarm string.quoted.double
#                   ^^^^^^^^^^^ source.swarm string.quoted.double
#                              ^ source.swarm string.quoted.double
#                               ^ source.swarm
#                                ^^^ source.swarm keyword.control.dictionary.def.end
#                                   ^ source.swarm keyword.operator.chain
>
>pure hello;
#^^^^ source.swarm keyword.other
#    ^ source.swarm
#     ^^^^^ source.swarm variable.parameter
#          ^ source.swarm keyword.operator.chain
>